test/src/tests-session-manager.ts

```diff
@@ -493,8 +493,7 @@ describe('SessionManager', () => {
                 );
             });
 
-            it('should NOT end session if session has not timed out', () => {
-                const now = new Date();
+            it('should NOT end a session if NOT enough time has passed in between sessions', () => {
```

Comment:
move this up to be right after  `should end a session if enough time has passed in between sessions`  for context consistency